salt/modules/state.py

```diff
@@ -599,6 +598,22 @@ def apply_(mods=None,
         .. code-block:: bash
 
             salt '*' state.apply test localconfig=/path/to/minion.yml
+
+    sync_mods
+        If specified, the desired custom module types will be synced prior to
+        running the SLS files:
+
+        .. code-block:: bash
+
+            salt '*' state.apply test sync_mods=states,modules
+            salt '*' state.apply test sync_mods=all
+
+        .. note::
+            This option is ignored when no SLS files are specified, as a
+            :ref:`highstate <running-highstate>` automatically syncs all custom
+            module types.
+
+        .. versionadded:: 2017.7.7,2018.3.3,Fluorine
```

Comment:
Ah, i just noticed this since you mentioned updating 2018.3.3 in another comment. The `2017.7.7` should be `2017.7.8`

Reply:
Done.